### PR TITLE
Week 7 Assignments

### PR DESCRIPTION
**Pull request by anugrahpaul2000**

### CODE REVIEW

File: Week-7/Assignment-1/index.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  reverse() {
+    if (this.head === null) {
+      console.log(`Linked List is Empty. \nNothing to Reverse`);
+      return;
+    }
+
+    this.current = this.head;
+    this.previous = null;
+    while (this.current !== null) {
+      this.next = this.current.next;
```
Comment (subhadip-pesto) [inline Week-7/Assignment-1/index.js]: can we have this next property local to reverse(). i think we can do this without storing it as a class property.

**Review by subhadip-pesto — CHANGES_REQUESTED** [review]

File: Week-7/Assignment-1/index.js (hunk 1 above)
Comment (anugrahpaul2000) [reply Week-7/Assignment-1/index.js]: yes definitely.

**Comment by anugrahpaul2000** [comment]
Assignment 6 in Progress.

**Comment by anugrahpaul2000** [comment]
@subhadip-pesto Assignments Done

**Review by subhadip-pesto — APPROVED** [review]

**Pull request merged**